docs/tutorials/configure/scuttlebot.md

```diff
@@ -3,7 +3,7 @@ title: "Configure a SCUTTLE Robot"
 linkTitle: "Configure a SCUTTLE Robot"
 weight: 15
 type: "docs"
-description: "Configure a SCUTTLE Robot on the Viam platform."
+description: "Configure a Scuttle robot on the Viam platform."
```

Comment:
```suggestion
description: "Configure a SCUTTLE robot on the Viam platform."
```
Is the description something a screen reader might read incorrectly, but that a person will never see?

Reply:
exactly